src/main/java/com/labsynch/labseer/utils/PropertiesUtilServiceImpl.java

```diff
@@ -968,4 +968,24 @@ public Integer getMaxAutoLotNumber() {
 		return this.maxAutoLotNumber;
 	}
 
+	public long checkForDryRunStandardizationDelay;
+
+	@Value("${client.cmpdreg.serverSettings.checkForDryRunStandardizationDelay}")
```

Comment:
This configuration sets the delay/how often roo checks to see if it should participate in an actively running dry run standardization.